net-dns/dnstop/files/dnstop-20140915-musl-fix.patch

```diff
@@ -0,0 +1,11 @@
+--- ./dnstop.c.orig
++++ ./dnstop.c
```

Comment:
Please avoid `.` and `.orig` and do either:
```suggestion
--- dnstop-20140915/dnstop.c
+++ dnstop-20140915/dnstop.c
```
or:
```suggestion
--- a/dnstop.c
+++ b/dnstop.c
```
Also, please add a reference in the patch to the bug that it fixes.

Policy reference:
https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto

Reply:
Thanks for the review. Should be fixed now. I've also included the upstream bug from alpine. Hope that's correct now.